add-screenshot/index.md

```diff
@@ -0,0 +1,28 @@
+## Adding screenshots to my game page
+
+There's a distinct possibility that screenshots are the single most vital element of your game pages, in terms of catching the attention of people who have already been enticed by your thumbnail, or who are visiting via a direct link to your game page. Many people will simply skip over a downloadable game with no screenshots.
+
+Even if your game is played in-browser, your game page should have some screenshots on it. A prospective player's interest might wane more quickly than the game loads, but a picture could capture their attention. And who knows, we might change the way browser games are served up in the future, so screenshots might become a gamer's only visual cue to the game's content.
+
+If your game is a work-in-progress, it's essential to post screenshots in order to get--and keep--people interested in your game. Screenshots are excellent proof of development for anyone interested in following along with your progress.
+
+Screenshots can even keep people who have already played your game coming back for more, having perhaps seen things in the images that they haven't yet seen in the game.
+
+One more thing about screenshots: You can actually upload images that aren't screenshots, too. But only post images that are relevant and your original work. Say you built your game's graphics out of clay; a photo of the models might make a cool "screenshot". Or maybe you built a custom arcade cabinet for your game, or some mockup box art; a picture of that would be perfect.
+
+### How to add screenshots
+
+To add screenshots, go to your [developer dashboard](http://gamejolt.com/dashboard/) and select a game from under "Your Games". You will be taken to the game's management page. Visit the "Media" tab and start adding images.
```

Comment:
"You will be taken to the game's management page." it's actually called the overview page... does that matter? should we continue to call it the management page? 

Side note - the media tab needs copy :) right now it just says blah blah blah